config/plugins/visualizations/pca/config/pca.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE visualization SYSTEM "../../visualization.dtd">
+<visualization name="PCA 3D Plot" embeddable="true">
+    <description>Principle Component Analysis</description>
+    <tags>
+        <tag>PCA</tag>
+        <tag>Eigenvector</tag>
+    </tags>
+    <data_sources>
+        <data_source>
+            <model_class>HistoryDatasetAssociation</model_class>
+            <test test_attr="ext">csv</test>
```

Comment:
It would be nice if this one can also work on tsv and tabular files.

Reply:
I added tsv, and tabular. When using tabular it treats the first row as headers i.e. column names.